schema/release-schema.json

```diff
@@ -823,7 +823,7 @@
         },
         "items": {
           "title": "Items contracted",
-          "description": "The goods, services, and any intangible outcomes in this contract. If the items contracted are identical to the items awarded, this field should be omitted.",
+          "description": "The goods, services and works in this contract. If the items contracted are identical to the items awarded, this field should be omitted.",
```

Comment:
Works and intangible outcomes seem like different concepts, but I don't know the reason that intangible outcomes are mentioned here but not elsewhere. @jpmckinney any ideas?

Reply:
The wording seems to have originated in https://github.com/open-contracting/standard/issues/105#issuecomment-59630318

Anyone, a service (i.e. an activity) is intangible, e.g. advisory services, so it's already covered. We shouldn't introduce a unique concept at the stage of contract items.

Reply:
Thanks, I agree. @odscrachel taking account of https://github.com/open-contracting/standard/pull/1530#discussion_r908510347, we should update this to:

```suggestion
          "description": "The goods, services or works in this contract. If the items contracted are identical to the items awarded, this field should be omitted.",
```